### PR TITLE
added validators for start_frame, stop_frane and chunk_size

### PR DESCRIPTION
**Pull request by azhavoro**

### CODE REVIEW

File: cvat/apps/engine/serializers.py [hunk 1]
```diff
@@ -173,12 +173,38 @@ class Meta:
...
 
+    # pylint: disable=no-self-use
+    def validate_chunk_size(self, value):
+        if not value > 0:
+            raise serializers.ValidationError('Chunk size must be a positive integer')
+        return value
+
+    # pylint: disable=no-self-use
+    def validate_start_frame(self, value):
+        if not value > 0:
```
Comment (nmanovic) [inline cvat/apps/engine/serializers.py]: value >= 0?

File: cvat/apps/engine/serializers.py [hunk 2]
```diff
@@ -173,12 +173,38 @@ class Meta:
...
+            raise serializers.ValidationError('Start frame must be a non negative integer')
+        return value
+
+    # pylint: disable=no-self-use
+    def validate_stop_frame(self, value):
+        if not value > 0:
```
Comment (nmanovic) [inline cvat/apps/engine/serializers.py]: What is the value for stop frame by default?

File: cvat/apps/engine/serializers.py (hunk 1 above)
Comment (azhavoro) [reply cvat/apps/engine/serializers.py]: My bad, it's PositiveIntegerField at the Data model that can be 0 and manual validation is not needed in this case.

File: cvat/apps/engine/serializers.py (hunk 2 above)
Comment (azhavoro) [reply cvat/apps/engine/serializers.py]: The same as above, I've deleted this validation.

**Pull request merged**